metis-normalization/src/main/java/eu/europeana/normalization/normalizers/DatesNormalizer.java

```diff
@@ -417,6 +421,7 @@ private void appendTimespanEntity(Document document, AbstractEdtfDate edtfDate,
                                    .map(InstantEdtfDate::getCentury).orElse(null);
     Integer endCentury = Optional.ofNullable(lastDay)
                                  .map(InstantEdtfDate::getCentury).orElse(null);
+
     // TODO: 25/07/2022 What if both are null, won't the 'for' loop below fail? Is there always at least one century?
```

Comment:
At this point, everything should be valid so that is not possible.
For a sanity check perhaps we can check for that case and throw an exception if that happens.
Or the date objects, as before, are by their instantiation validated.